core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/imagecatalog/ImageToImageV4ResponseConverter.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.cloudbreak.converter.v4.imagecatalog;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.imagecatalog.responses.ImageV4Response;
+import com.sequenceiq.cloudbreak.cloud.model.catalog.Image;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+
+@Component
+public class ImageToImageV4ResponseConverter extends AbstractConversionServiceAwareConverter<Image, ImageV4Response> {
+
+    @Override
+    public ImageV4Response convert(Image source) {
```

Comment:
At least stackDetails conversion is missing. It was not needed for freeipa, it is the reason it was not implemented by me. Please make sure all ImageV4Response paremeters are set up.

Reply:
The conversation is extended with the stackDetails param.